### PR TITLE
Report console errors from displaying tags

### LINKED ISSUE #15: Report console errors from displaying tags

### PR DESCRIPTION
**Pull request by d3ming**
Also add a ton of integration tests to bring integration coverage to 67%, unit test coverage to 41%

Closes #15

@tmillner - with this we will report console logs

### CODE REVIEW

File: tagcompare/capture.py [hunk 1]
```diff
@@ -50,42 +52,61 @@ def _write_html(tag_html, output_path):
...
         LOGGER.debug("Skipping existing captures %s", default_pb.path)
+        return None
+
+    driver = __start_webdriver_capture(capabilities,
+                                       build=pathbuilder.build,
+                                       name=pathbuilder.config)
+    try:
+        tag_html = tags_per_campaign[tagsize]
+
+        # TODO: log browser errors
```
Comment (tmillner) [inline tagcompare/capture.py]: wouldn't this be logged on L:125?

File: tagcompare/test/test_capture.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from tagcompare import capture
+from tagcompare import settings
+from tagcompare import output
+
+
+@pytest.mark.integration
+def test_capture_configs():
```
Comment (tmillner) [inline tagcompare/test/test_capture.py:9]: nice!

File: tagcompare/webdriver.py [hunk 3]
```diff
@@ -12,42 +12,70 @@
...
+def check_browser_logs(driver):
+    """
+    Checks browser for errors, returns a list of errors
+    :param driver:
+    :return:
+    """
+    browserlogs = driver.get_log('browser')
+    errors = []
+    for entry in browserlogs:
+        if entry['level'] == 'SEVERE':
```
Comment (tmillner) [inline tagcompare/webdriver.py:58]: Wonder if setting this lower to warning will provide similar value without being too verbose.

File: tagcompare/capture.py (hunk 1 above)
Comment (d3ming) [inline tagcompare/capture.py]: Yup removed

File: tagcompare/webdriver.py (hunk 3 above)
Comment (d3ming) [inline tagcompare/webdriver.py:58]: Gonna stick to errors for now, it'll be trivial to add more to this later

**Pull request merged**